mne_bids/mne_bids.py

```diff
@@ -73,6 +74,11 @@ def _channels_tsv(raw, fname, verbose):
         The data as MNE-Python Raw object.
     fname : str
         Filename to save the channels.tsv to.
+    overwrite : bool
+        Defaults to False.
+        Whether to overwrite the existing file.
+        If the file exists already but overwrite == False an `OSError` will be
+        raised.
```

Comment:
I think `L80-81` is not necessary. We don't document the errors in the docstring. Just make sure the different cases and usecases are tested and exposed in examples.

Also, I'd swap the order of `L78` and `L79`